.codeclimate.yml

```diff
@@ -12,3 +13,8 @@ checks:
   method-complexity:
     config:
       threshold: 8
+plugins:
+ duplication:
+   enabled: true
+   config:
+     count_threshold: 3
```

Comment:
question: this doesn't seem to follow the docs here: https://docs.codeclimate.com/docs/duplication

Are there docs somewhere else I'm not seeing?

Reply:
nevermind, there are more docs in the plugin repo: https://docs.codeclimate.com/docs/duplication